nextflow_schema.json

```diff
@@ -38,6 +38,116 @@
+                },
+                "plasmidfinder_database": {
+                    "type": "string",
+                    "enum": ["All", "gram_positive", "enterobacteriales"],
+                    "description": "The database type to use for plasmidfinder {gram_positive, enterobacteriales}. Defaults to using all available database types to search for plasmids. [All]."
+                },
+                "mlst_scheme": {
+                    "type": "string",
+                    "description": "Specify scheme name, visit https://github.com/tseemann/mlst/tree/master/db/pubmlst for supported scheme genera available. [None]",
+                    "default": "Automatic"
+                }
+            }
+        },
+        "additional_settings": {
+            "title": "Additional Settings",
+            "type": "object",
+            "description": "For advanced changes to staramr",
+            "properties": {
+                "genome_size_lower_bound": {
+                    "type": "integer",
+                    "description": "The lower bound for our genome size for the quality metrics [Default 4000000]",
+                    "default": 4000000,
+                    "minimum": 1
+                },
+                "genome_size_upper_bound": {
+                    "type": "integer",
+                    "description": "The upper bound for our genome size for the quality metrics [Default 6000000].",
+                    "default": 6000000,
+                    "minimum": 1
+                },
+                "minimum_N50_value": {
+                    "type": "integer",
+                    "description": "The minimum N50 value for the quality metrics [Defaults 10000]",
+                    "default": 10000,
+                    "minimum": 1
+                },
+                "minimum_contig_length": {
+                    "type": "integer",
+                    "description": "The minimum contig length for the quality metrics [Default 300 bp]",
+                    "default": 300,
+                    "minimum": 1
+                },
+                "unacceptable_number_contigs": {
+                    "type": "integer",
+                    "description": "The minimum, unacceptable number of contigs which are equal to or above the minimum contig length for our quality metrics [Default 1000]",
+                    "default": 1000,
+                    "minimum": 1,
+                    "maximum": 500000
+                },
+                "pid_threshold": {
+                    "type": "integer",
+                    "description": "BLAST percent identity threshold [Default 98]",
+                    "default": 98,
+                    "minimum": 1,
+                    "maximum": 100
+                },
+                "percent_length_overlap_plasmidfinder": {
+                    "type": "integer",
+                    "description": "The percent length overlap for plasmidfinder results [Default 60.0]",
```

Comment:
Could you remove the decimal point from `60.0`. I know it's the same value, but if we are only allowing integer values for the pipeline then including a decimal point in the default value may make it confusing. Could you change other defaults like this too?